airflow/operators/dagrun_operator.py

```diff
@@ -15,88 +15,14 @@
 # KIND, either express or implied.  See the License for the
 # specific language governing permissions and limitations
 # under the License.
+"""This module is deprecated. Please use `airflow.operators.dagrun`."""
 
-import datetime
-from typing import Dict, Optional, Union
-from urllib.parse import quote
+import warnings
 
-from airflow.api.common.experimental.trigger_dag import trigger_dag
-from airflow.models import BaseOperator, BaseOperatorLink, DagRun
-from airflow.utils import timezone
-from airflow.utils.decorators import apply_defaults
-from airflow.utils.types import DagRunType
+# pylint: disable=unused-import
+from airflow.operators.dagrun import TriggerDagRunOperator # noqa
 
-
-class TriggerDagRunLink(BaseOperatorLink):
```

Comment:
You missed moving this class as well as operator_extra_links in the `TriggerDagRunOperator`

Reply:
Got it. Just noticed. I guess I didn't read the conflict properly although when I was working on it the particular `class TriggerDagRunLink(BaseOperatorLink)` didn't exist in the file. I'll fix this now.

Please correct me if I'm wrong, so to fix this, I should finish working on the file, fetch/rebase, then push again. Right?

Reply:
I have pushed with the corrections. @turbaszek please have a look.

Apologies for the previous commit. I messed up with the rebase.